apps/bluetooth/js/views/pair_view.js

```diff
@@ -68,30 +74,23 @@ var Pairview = {
         this.passkey.textContent = this._passkey;
         this.comfirmationItem.hidden = false;
         this.pinInputItem.hidden = true;
-        this.passkeyInputItem.hidden = true;
         break;
 
       case 'pincode':
         this.pinInputItem.hidden = false;
         this.comfirmationItem.hidden = true;
-        this.passkeyInputItem.hidden = true;
         this.pinInput.focus();
         break;
-
-      case 'passkey':
-        this.passkeyInputItem.hidden = false;
-        this.comfirmationItem.hidden = true;
-        this.pinInputItem.hidden = true;
-        this.passkeyInput.focus();
-        break;
     }
   },
 
-  init: function pv_init(mode, method, device, passkey) {
-    this._pairMode = mode;
+  init: function pv_init(method, evt) {
```

Comment:
Let's use `options` as the name of the parameter. Please also add jsdoc describing the properties in the options object.